src/pages/docs/sdk/parameters/set-data.md

```diff
@@ -32,7 +32,7 @@ export default () => (
 
 <Microlink url='https://instagram.com/p/Bt6EMQhHHIr/' setData={{title: 'SENTRY ACTIVATED' }}/>
 
-When an `object` is passed to `setData`, the custom data provided will merged with the original data.
+When an `object` is passed to `setData`, the custom data provided will be merged with the original data.
 
 If you need a more granular control, you can pass a `function`.
 
```

Comment:
@Kikobeats I don't think the caption at the bottom of this file is right:

> Skip internal fetch providing a function as setData.

It doesn't skip internal fetching right? It just uses the prop's function for passing the data into the card instead of merging it

Reply:
That's right, it's outdated